packages/server/src/course/course.controller.ts

```diff
@@ -91,6 +93,13 @@ export class CourseController {
     queue.staffList.push(user);
     await queue.save();
 
+    const UCM = await UserCourseModel.findOne({ where: { user, courseId } });
+    await EventModel.create({
+      time: new Date(),
+      eventType: EventType.TA_CHECKED_IN,
+      user: UCM,
```

Comment:
what if we stored the userId and courseId directly in the eventmodel instead? UserCourseModels get destroyed and recreated on login so storing a reference to the ucm may get fucked when someone logs in again. It would also be less querying this way.

Reply:
are you saying just as numbers and not as relations?

Reply:
i just mean a relation to user and a relation to course, but not a relation to UserCourse